src/main.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+function Person(name, dateOfBirth, amount) {
+    this.name = name;
+    this.dateOfBirth = dateOfBirth.split('.').reverse();
+    this.amount = amount;
+
+    let [year, month, day] = this.dateOfBirth;
+
+    this.age = function getAge() {
+        const currentYear = new Date().getFullYear();
+        const currentMonth = new Date().getMonth() + 1;
+        const currentDay = new Date().getDate();
+        if (currentMonth < month || currentMonth === month && currentDay < day) {
+            return currentYear - year + 1;
+        }
+        return currentYear - year;
```

Comment:
you can create a variable for `currentYear - year` and return variable or variable + 1;